pkg/kv/kvclient/rangefeed/rangefeed.go

```diff
@@ -267,6 +268,8 @@ func (f *RangeFeed) Close() {
 // will be reset.
 const resetThreshold = 30 * time.Second
 
+var useMuxRangeFeed = util.ConstantWithMetamorphicTestBool("use-mux-rangefeed", false)
```

Comment:
do you want to add something that fires when we flip the cluster setting to true? At that point, this won't have an effect any longer.

Reply:
We don't really have a cluster setting; enabling it is left to the caller;
when we add cluster setting, presumably we can remove with rangefeed option.. and remove this metamorphic constant then.